play-json/jvm/src/main/scala/play/api/libs/json/EnvWrites.scala

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import java.time.format.DateTimeFormatter
+import java.time.temporal.Temporal
+import java.time.{
+  Instant,
+  LocalDate,
+  LocalDateTime,
+  OffsetDateTime,
+  ZoneOffset,
+  ZonedDateTime,
+  ZoneId
+}
+
+import com.fasterxml.jackson.databind.JsonNode
+
+import play.api.libs.json.jackson.JacksonJson
+
+trait EnvWrites {
```

Comment:
For the JVM backend, the previous Jackson impl is kept, with the corresponding instances of the typeclasses `Writes` and `Reads`